CHANGELOG.md

```diff
@@ -44,6 +44,7 @@ Every module contains its own CHANGELOG.md. Please refer to the module you are i
 
 * (baseapp) [#20291](https://github.com/cosmos/cosmos-sdk/pull/20291) Simulate nested messages.
 * (client/keys) [#21829](https://github.com/cosmos/cosmos-sdk/pull/21829) Add support for importing hex key using standard input.
+* (server/v2) [#22715](https://github.com/cosmos/cosmos-sdk/pull/22941) Add custom HTTP handler for grpc-gateway that removes the need to manually register grpc-gateway services.
```

Comment:
Could oyu move this to server/v2/CHANGELOG.md? (We don't need the server/v2 prefix then)

Reply:
done in [add changelog to server/v2](https://github.com/cosmos/cosmos-sdk/pull/22941/commits/f0c3e47c1946c11cd962e559a7c754250b4e0e6a)! 

also question: given this broke the `grpcgateway.New` function, should i add a line to the API Breaking section?

Reply:
it's beta so it is fine as is 👍🏾